packages/enhanced/src/lib/container/ContainerPlugin.ts

```diff
@@ -164,6 +164,13 @@ class ContainerPlugin {
       compiler.options.output.enabledLibraryTypes.push(library.type);
     }
 
+    new compiler.webpack.EntryPlugin(
+      compiler.options.context || '',
+      federationRuntimePluginInstance.entryFilePath,
+      {
+        name,
```

Comment:
entry plugin with same name merges the two entry together, yes?

Reply:
yes, the init federation first and the second is export get and init function